### PR TITLE
Liu 220 - EnvironmentVariable DROP

### PR DESCRIPTION
**Pull request by pritchardn**
Adds a new generic component for storing globally available parameters.
This should be a minimal edition, some tests however may need re-working from the looks of things however.

### CODE REVIEW

**Comment by james-strauss-uwa** [comment]
If it is a hassle to get testcafe running locally, I could look into the reason for the test failures.

**Comment by pritchardn** [comment]
Oops, many thanks for picking up on that one

File: static/template.palette [hunk 1]
```diff
@@ -825,6 +825,32 @@
             "inputPorts": [],
             "readonly": true
         },
+        {
+            "category": "EnvironmentVars",
```
Comment (james-strauss-uwa) [inline static/template.palette:829]: Sorry, I think I changed this to "EnvironmentVariables" in Eagle.ts

That might have been the cause of the conflicts with the master branch.

**Review by james-strauss-uwa — APPROVED** [review]
I'll just change "EnvironmentVars" to "EnvironmentVariables", then it's ready to merge.

**Pull request merged**